File: packages/ios/Sources/CivUI/CivFileList.swift
// CivUI — CivFileList for SwiftUI
// Read-only display of uploaded files for review/confirmation pages.

import SwiftUI

public struct FileListItem {
    public var name: String
    public var size: Int
    public var url: String?
    public var type: String?

    public init(name: String, size: Int, url: String? = nil, type: String? = nil) {
        self.name = name
        self.size = size
        self.url = url
        self.type = type
    }
}

public struct CivFileList: View {
    public var files: [FileListItem]
    public var label: String

    public init(
        files: [FileListItem] = [],
        label: String = ""
    ) {
        self.files = files
        self.label = label
    }

    public var body: some View {
        EmptyView() // TODO: Implement
    }
}

File: packages/android/src/main/kotlin/gov/civui/components/CivFileList.kt
// CivUI — CivFileList for Jetpack Compose
// Read-only display of uploaded files for review/confirmation pages.

package gov.civui.components

import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier

data class FileListItem(
    val name: String = "",
    val size: Int = 0,
    val url: String? = null,
    val type: String? = null,
)

@Composable
fun CivFileList(
    files: List<FileListItem> = emptyList(),
    label: String = "",
    modifier: Modifier = Modifier,
) {
    // TODO: Implement
}
